README.md

```diff
@@ -137,6 +137,14 @@ See [CHANGELOG.md](CHANGELOG.md)
 > However, the API is based on the Principle of Least Privilege, and exposes only a limited set of methods.
 > We aren't quite there yet, but we envision Talos being a great place for the application of [control theory](https://en.wikipedia.org/wiki/Control_theory) in order to provide a self-healing platform.
 
+**How is Talos different than CoreOS/RancherOS/Linuxkit?**
+
+> Talos does a few things differently.
+> First, Talos is not meant to be a container based Linux distribution for general use.
+> It is targeted for Kubernetes and cannot be used for anything other than that.
+> Another differentiator is that Talos does not provide host-level access.
+> Interaction with the OS is done through an API.
```

Comment:
Talos is integrated with Kubernetes and cannot be deployed without it. It is not meant to be a general use Linux distribution. 
Talos does not provide host level access ( no shell, ssh, etc ). Interaction with the OS is done through the API via `osctl`.

Thoughts?

Reply:
Much better :)